### PR TITLE
Add ansys-mapdl-reader

### PR DESCRIPTION
**Pull request by akaszynski**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.
- [x] Get it working.

### CODE REVIEW

**Comment by akaszynski** [comment]
Hello,

Is there anything else I need to do to get this merged?

File: recipes/ansys-mapdl-reader/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cython
+    - python
```
Comment (xhochy) [inline recipes/ansys-mapdl-reader/meta.yaml:21]: This should only be in host
```suggestion
```

**Comment by akaszynski** [comment]
Pinging @xhochy for guidance regarding mergability.

I know you guys are busy and I really appreciate your work maintaining this!

**Comment by akaszynski** [comment]
Thanks!

**Pull request merged**